packages/contracts/src/acala/index.ts

```diff
@@ -0,0 +1,42 @@
+import { calcEthereumTransactionParams } from "@acala-network/eth-providers";
+import { ethers, network } from "hardhat";
+
+import type { HardhatRuntimeEnvironment } from "hardhat/types";
+import type { DeployOptions } from "hardhat-deploy/types";
+
+export async function isAcala() {
+  return [595].includes(network.config.chainId!);
```

Comment:
Why the includes and not an `===`?

Reply:
because this array is suposed to include more networks later on. 595 is the mandala testnet. this function will include karura and acala as well